### PR TITLE
Add Throws of FabanClientBadRequestException in all the Commands

### PR DESCRIPTION
**Pull request by VincenzoFerme**
As per the title

### CODE REVIEW

File: benchflow-faban-client/src/main/java/cloud/benchflow/faban/client/commands/RunInfoCommand.java [hunk 1]
```diff
@@ -66,9 +66,10 @@ private RunInfo runInfo(FabanClientConfig fabanConfig)
         throw new FabanClientBadRequestException("Illegal runInfo request");
       }
 
+      //Handle generic HTTP exceptions (TODO: determine the expected HTTP status from Faban, and validate we get that one)
       //TODO: check that the call to .handleEntity(..) actually returns the expected string
```
Comment (jeppe-style) [inline benchflow-faban-client/src/main/java/cloud/benchflow/faban/client/commands/RunInfoCommand.java]: probably should be referring to `.handleResponse`

File: benchflow-faban-client/src/main/java/cloud/benchflow/faban/client/commands/StatusCommand.java [hunk 2]
```diff
@@ -61,9 +61,9 @@ private RunStatus status(FabanClientConfig fabanConfig)
         throw new FabanClientBadRequestException("Illegal status request");
       }
 
+      //Handle generic HTTP exceptions (TODO: determine the expected HTTP status from Faban, and validate we get that one)
       //TODO: check that the call to .handleEntity(..) actually returns the expected string
```
Comment (jeppe-style) [inline benchflow-faban-client/src/main/java/cloud/benchflow/faban/client/commands/StatusCommand.java]: probably should be referring to `.handleResponse`

Comment (VincenzoFerme) [reply benchflow-faban-client/src/main/java/cloud/benchflow/faban/client/commands/StatusCommand.java]: Fixed in https://github.com/benchflow/benchflow/pull/547/commits/810d57c4e6381d5b7d63add3419e13a657ce21f7

File: benchflow-faban-client/src/main/java/cloud/benchflow/faban/client/commands/RunInfoCommand.java (hunk 1 above)
Comment (VincenzoFerme) [reply benchflow-faban-client/src/main/java/cloud/benchflow/faban/client/commands/RunInfoCommand.java]: Fixed in https://github.com/benchflow/benchflow/pull/547/commits/810d57c4e6381d5b7d63add3419e13a657ce21f7

**Review by jeppe-style — APPROVED** [review]

**Pull request merged**